tests/preprocess/incident_types/test_mapper.py

```diff
@@ -1,14 +1,19 @@
 import unittest
+import os
 
 from incident_types import processor
 from incident_types.incident_types_d import IncidentType, replacements
 from report_data import ReportData
 
+# use for filepath relative from this file
+dir_path = os.path.dirname(os.path.realpath(__file__))
 
 class TestIncTypeMapper(unittest.TestCase):
     def test_maps(self):
         """Ensure replacements are made as expected."""
-        report_df = ReportData().get_raw_report_data()
+
+        in_file_path = os.path.join(dir_path, '../../../preprocess/data/data-sensitive.csv')
+        report_df = ReportData(in_file_path=in_file_path).get_raw_report_data()
```

Comment:
Now that we have the absolute paths in `ReportData` we can probably remove this.